DOC-12531-Feedback on set flush_param

supritha-kumar opened the pull request:
Changed "The cbepctl flush_param exp_pager_stime command sets the time interval for disk cleanup " to 

"The `cbepctl flush_param exp_pager_stime` command sets the time interval to scan for expired items, and erase them from memory and disk."

https://jira.issues.couchbase.com/browse/DOC-12531

supritha-kumar committed:
DOC-12531-Feedback on set flush_param

Changed "The cbepctl flush_param exp_pager_stime command sets the time interval for disk cleanup " to 

"The `cbepctl flush_param exp_pager_stime` command sets the time interval to scan for expired items, and erase them from memory and disk."

supritha-kumar committed:
Update set-flush_param.adoc

changed disk cleanup to asynchronous expiry

Pallavi-Janardhan commented on modules/cli/pages/cbepctl/set-flush_param.adoc:
We can remove the comma after items and before "and erase".

Pallavi-Janardhan reviewed (approved):
LGTM

supritha-kumar committed:
Updated set-flush_param.adoc

removed comma from "scan for expired items and erase them from memory and disk"

Pallavi-Janardhan reviewed (approved):
LGTM
